firmware/kivipallur_firmware_3.0/Core/Src/main.c

```diff
@@ -193,10 +193,17 @@ int32_t PIDcontrol(MotorControl* control, int16_t position){
 uint16_t pwmData[20];
 
 void Thrower_Send (uint16_t command){
+	if (command > 6000, command < 1) {
+		return 0;
+	}
+	return command;
```

Comment:
* if statement conditions should not be separated with a comma.
* 0 should not be used as a PWM duty cycle. Constantly low signal would be detected as lost signal for the ESC and the ESC would need to rearmed to work.
* `Thrower_Send` is not changing duty cycle value and the return value is also not used to change duty cycle.